### ISSUE TITLE
Make it easier to display a list of indexed items

### ISSUE THREAD

**Issue by 3lvis** [3lvis/DATASource#16]
Implementing 

```objc
- (NSArray *)sectionIndexTitlesForDataSource:(UITableView *)tableView;

- (NSInteger)dataSource:(UITableView *)tableView
sectionForSectionIndexTitle:(NSString *)title
                atIndex:(NSInteger)index;
```

should be way easier.

**Comment by 3lvis** [comment]
http://nshipster.com/uilocalizedindexedcollation/

**Comment by 3lvis** [comment]
Fixed on #20
